src/main/resources/templates/email/placement-updated-week-12/v1.0.0.html

```diff
@@ -47,8 +49,9 @@ <h2>Content</h2>
     Further details on the specialist area of your placement will be confirmed by your employing organisation. This is because we are only able to confirm details that relate specifically to your college recognised curriculum.
   </p>
   <p>
-    An example of this would be If you have preference a specific post with your Training Programme Director but does not relate to sub-specialty training. This may be a job in a specific location or department
+    An example of this would be if you have a preference for a specific post within your Training Programme that does not relate to sub-specialty training. This may be a job in a specific location or department. Please confirm this with your Training Programme Director.
   </p>
+  <p><b>We are currently piloting the Placement confirmation service. You may come across a series of questions about this within the TIS Self Service app. We appreciate your feedback to help us improve the process and make it easy for you to access the necessary information.</b></p>
```

Comment:
I think we're trying to standardise on 'TIS Self-Service' (with hyphen) - though we probably need to go through all the templates to confirm they are consistent!

Reply:
Updated all the TIS Self Service wording with a hyphen in the Placement notification